### DISCUSSION TITLE
如何自定义单元格内容

### DISCUSSION THREAD

**Discussion by wangdp96** [antvis/S2#2850] [General]
1.X版本，需要将单元格内的字符串链接转化为图片展示在表格，怎么做？官网没找到案例
<img src="***" />以及其他html标签的使用

**Comment by lijinke666** [comment]
https://s2-v1.antv.antgroup.com/examples/custom/custom-cell/#data-cell

不支持 html, 表格是 Canvas, 不是 DOM
